frontend/sw360-portlet/src/main/resources/content/Language.properties

```diff
@@ -177,7 +179,7 @@ clearing.state=Clearing State
 ClearingState=New: &#10Sent to fossology: &#10Under Clearing: &#10Report available: &#10Approved:
 clearing.status=Clearing Status
 clearing.summary=Clearing summary
-clearing.team=Clearing team
+clearing.team=Clearing Team
```

Comment:
This is because Vietnamese translation for  `clearing.team`  and `requesting.user` is already present in `Language_vi.properties`,
I don't know Vietnamese, So I can't `capitalize` Vietnamese in `Language_vi.properties`,